README.md

```
@@ -23,6 +23,17 @@ python manage.py runserver 0.0.0.0:8000
 
 Visit [`/health`](http://localhost:8000/health) for a readiness probe.
 
+## Endpoints
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)